src/org/exist/dom/persistent/ElementImpl.java

```diff
@@ -1734,17 +1734,6 @@ private XmldbURI calculateBaseURI() {
     }
 
     @Override
-    public boolean isSameNode(final Node other) {
```

Comment:
without knowing too much of the details, I did not expect this to be removed.  Where is this 'detail' check done now?